tests/tasks/lib/test_sst.py

```diff
@@ -0,0 +1,149 @@
+import os
```

Comment:
Nice format for a test. Seems easily extendable to other tasks with a few minor changes (passing in task name, removing task name from function name).

Reply:
> removing task name from function name

Addressed in 021944d, thanks!